### PR TITLE
2FA update

### LINKED ISSUE #25: Authy vs Google authenticator
In section 2.1.2 (Enable two-factor authentication on your Github account) the guidance suggests Authy or Google authenticator. However Authy seems better for when the users device changes. It might be worth suggesting Authy over Google Auth? Or if there is a way for the user to reset their credentials (now or in the future) to add guidance on how to reset in this section.

### PR DESCRIPTION
**Pull request by calumabarnett**
Small update to the guidance on 2FA. Recommends use of Authy as opposed to Google Authenticator. Fixes #25.

### CODE REVIEW

File: 02-start.Rmd [hunk 1]
```diff
@@ -2,24 +2,31 @@
...
 
-You are given the option of doing this using SMS, or an authentication app such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en) . Choose whichever you prefer.
+You are given the option of doing this using SMS, or an authentication app, such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en). We recommend that you use __Authy__ for the following reasons:
 
-For step-by-step instructions, see the Github guidance for [authentication apps](https://help.github.com/articles/configuring-two-factor-authentication-via-a-totp-mobile-app/) or [SMS authentication](https://help.github.com/articles/configuring-two-factor-authentication-via-text-message/).
+1.  You can sync 2FA tokens across multiple devices, including mobiles, tablets and computers.
+2.  You can create encrypted recovery backups in case you lose your device.
+3.  You can use Touch ID, PIN codes and passwords to protect access to your 2FA tokens.
+4.  You can still access secure websites when you are offline.
```
Comment (davidread) [inline 02-start.Rmd]: It's put a bit abstractly. How about: SMS codes don't work when you have no phone signal

File: 02-start.Rmd [hunk 2]
```diff
@@ -39,28 +46,28 @@ If you wish to access the platform from another formal HMG technology solution (
 
 You must not access the platform from a non-MOJ device, such as a personal mobile phone or laptop.
 
-When you first access your platform account, you will be asked to set up additional two factor authentication for the platform itself.  You must do this using an authentication app on your personal or work phone (such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en)).  There is no SMS option.
+When you first access your platform account, you will be asked to set up additional two-factor authentication for the platform itself.  You must do this using an authentication app, as no SMS option is available. For the reasons described in Section \@ref(two-factor-authentication), we recommend that you use [Authy](https://authy.com/download/).
```
Comment (davidread) [inline 02-start.Rmd]: "on your personal or work phone" seems useful reassurance that people are allowed to use their personal phone, or have I missed something?

File: 13-errors.Rmd [hunk 3]
```diff
@@ -78,7 +78,7 @@ and then read it in using e.g.
 
 Note, it's best to avoid using `aws.s3` directly, see [here](https://github.com/moj-analytical-services/platform_user_guidance/blob/master/05-errors.Rmd#unable-to-access-data-using-awss3-package)
 
-## Two Factor Authentication problems
+## Two Factor Authentication problems {#two-factor-auth-issues}
 
 Two factor authentication is critical to the security of the platform.  We have opted to use smartphone based 2FA apps due to the expense of giving out hardware tokens like the RSA device you use to log in to DOM1.
```
Comment (davidread) [inline 13-errors.Rmd]: While you're in this doc, you might add that we don't allow SMS code due to their susceptibility to interception.

File: 02-start.Rmd (hunk 1 above)
Comment (calumabarnett) [reply 02-start.Rmd]: I agree. Updated in c81d959.

File: 02-start.Rmd (hunk 2 above)
Comment (calumabarnett) [reply 02-start.Rmd]: Clarified this in 596f6a1.

File: 13-errors.Rmd (hunk 3 above)
Comment (calumabarnett) [reply 13-errors.Rmd]: Added in 071b362.

**Pull request merged**